apps/bsd/src/AppRoot.tsx

```diff
@@ -247,6 +257,8 @@ const App: React.FC = () => {
       await fetch('/scan/zero', {
         method: 'post',
       })
+      await patchConfig({ markThresholdOverrides: null })
```

Comment:
oh I missed this in the other PR: I feel like this threshold override reset should be part of the `/scan/zero` API backend, so there's no chance of doing one without the other.